src/status_im/chat/models/input.cljs

```diff
@@ -134,8 +133,7 @@
                                             :content      (cond-> {:chat-id current-chat-id
                                                                    :text    input-text}
                                                             message-id
-                                                            (assoc :response-to old-message-id
-                                                                   :response-to-v2 message-id)
+                                                            (assoc :response-to-v2 message-id)
```

Comment:
Does this `v2` now tie into the status-protocol-go's `v1`? Maybe we should get rid of this `v2` naming at some point?

Reply:
Yes, I decided not to make any protocol change in this PR to keep the change surface small, but given that this is merged, we can just rename  `-v2` to `-v1` or non-namespaced